src/app/wallets/index.ts

```diff
@@ -23,3 +23,9 @@ export const listWalletsByAccountId = async (
 ): Promise<Wallet[] | RepositoryError> => {
   return WalletsRepository().listByAccountId(accountId)
 }
+
+export const getAllWalletIds = async (
```

Comment:
I would call it like this for consistency

```suggestion
export const listWalletIds = async (
```